book/src/storage/storage-functions.md

```diff
@@ -0,0 +1,242 @@
+
+```move
+// File: sui-framework/sources/transfer.move
+public fun transfer<T: key>(obj: T, recipient: address);
+```
+
+In the next example, you can see how it can be used in a module that defines and sends an object to the transaction sender.
+
+```move
+module book::transfer_to_sender {
+
+    /// A struct with `key` is an object. The first field is `id: UID`!
+    public struct AdminCap has key { id: UID }
+
+    /// Init function is a special function that is called when the module
+    /// is published. It is a good place to create application objects.
+    fun init(ctx: &mut TxContext) {
+        // Create a new `AdminCap` object, in this scope.
+        let admin_cap = AdminCap { id: object::new(ctx) };
+
+        // Transfer the object to the transaction sender.
+        transfer::transfer(admin_cap, ctx.sender());
+
+        // admin_cap is gone! Can't be accessed anymore.
+    }
+
+    /// Transfers the `AdminCap` object to the `recipient`. Thus, the recipient
+    /// becomes the owner of the object, and only they can access it.
+    public fun transfer_admin_cap(cap: AdminCap, recipient: address) {
+        transfer::transfer(cap, recipient);
+    }
+}
+```
+
+When the module is published, the `init` function will get called, and the `AdminCap` object which we created there will be _transferred_ to the transaction sender. The `ctx.sender()` function returns the sender address for the current transaction.
+
+Once the `AdminCap` has been transferred to the sender, let's say `0xa11ce`, the sender, and only the sender, will be able to access the object. The object is now _account owned_.
+
+> Account owned objects are a subject to _true ownership_ - only the account owner can access them. This is a fundamental concept in the Sui storage model.
+
+Let's extend the example with a function that uses `AdminCap` to authorize a mint of a new object and its transfer to another address:
+
+```move
+/// Some `Gift` object that the admin can `mint_and_transfer`.
+public struct Gift has key { id: UID }
+
+/// Creates a new `Gift` object and transfers it to the `recipient`.
+public fun mint_and_transfer(
+    _: &AdminCap, recipient: address, ctx: &mut TxContext
+) {
+    let gift = object::new(ctx);
+    transfer::transfer(gift, recipient);
+}
+```
+
+The `mint_and_transfer` function is a public function that "could" be called by anyone, but it requires an `AdminCap` object to be passed as the first argument. Without it, the function will not be callable. This is a simple way to restrict access to privileged functions called _[Capability](./../programmability/capability.md)_. Because the `AdminCap` object is _account owned_, only `0xa11ce` will be able to call the `mint_and_transfer` function.
```

Comment:
```suggestion
The `mint_and_transfer` function is a public function that "could" be called by anyone, but it requires an `AdminCap` object to be passed as the first argument by reference. Without it, the function will not be callable. This is a simple way to restrict access to privileged functions called _[Capability](./../programmability/capability.md)_. Because the `AdminCap` object is _account owned_, only `0xa11ce` will be able to call the `mint_and_transfer` function.
```